quickevent/app/quickevent/plugins/Event/src/services/emmaclient.cpp

```diff
@@ -443,14 +443,18 @@ void EmmaClient::exportStartListRacomTxt()
 		lastId = id;
 		int si = q2.value("runs.siId").toInt();
 		int startTime = q2.value("runs.startTimeMs").toInt();
+		bool startTimeCardNull = q2.value("runs.startTimeMs").isNull();
 		int startTimeCard = q2.value("cards.startTime").toInt();
-		if (startTimeCard == 61166)
+		if (startTimeCard == 61166 || startTimeCardNull)
 			startTimeCard = 0;
 		QString name = q2.value("competitors.lastName").toString() + " " + q2.value("competitors.firstName").toString();
 		QString clas = q2.value("classes.name").toString();
```

Comment:
class

Reply:
Please explain to me ?

Reply:
`clas` has no meaning, I guess, that you meant `class` and `clas` is just typo.

Reply:
class is reserved word :(

Reply:
Yes, sorry :)